# HG changeset patch
# User Rik <rik@octave.org>
# Date 1399906601 25200
#      Mon May 12 07:56:41 2014 -0700
# Node ID 38925538ec14a421b133e5b2d89e59dbd2842860
# Parent  9eb72fa5f8b5b2fb19e3f49261abebe1d7034929
datevec.m: Change isrow to iscolumn in cset 9eb72fa5f8b5.

* datevec.m: Change isrow to iscolumn in cset 9eb72fa5f8b5.

diff --git a/scripts/time/datevec.m b/scripts/time/datevec.m
--- a/scripts/time/datevec.m
+++ b/scripts/time/datevec.m
@@ -143,17 +143,17 @@ function [y, m, d, h, mi, s] = datevec (
         if (! found)
           error ("datevec: DATE not parsed correctly with given format");
         endif
       endfor
     endif
 
   else   # datenum input
 
-    if (! isrow (date))
+    if (! iscolumn (date))
       date_sz = size (date);
       do_resize = true;
     endif
     date = date(:);
 
     ## Move day 0 from midnight -0001-12-31 to midnight 0000-3-1
     z = floor (date) - 60;
     ## Calculate number of centuries; K1 = 0.25 is to avoid rounding problems.
